curriculum/challenges/english/25-front-end-development/workshop-accessibility-quiz/614ccc21ea91ef1736b9b578.md

```diff
@@ -0,0 +1,67 @@
+---
+id: 614ccc21ea91ef1736b9b578
+title: Step 1
+challengeType: 0
+dashedName: step-1
+demoType: onLoad
+---
+
+# --description--
+
+For this workshop, you have been provided with a basic boilerplate.
+
+Start this accessibility journey by providing a `lang` attribute to your `html` element. This will assist screen readers in identifying the language of the page.
+
+Also, remember to link your stylesheet to the page.
+
+# --hints--
+
+You should give the `html` element a `lang` attribute. _Hint: You can use the value `en` for English._
+
+```js
+assert.match(code, /<html\s+lang=('|")[\w\-]+?\1\s*>/i);
+// TODO: This should/could be fixed in the builder.js
+// assert.notThrow(Intl.getCanonicalLocales(document.querySelector('html').lang));
+```
+
+You should have a `link` element in your `head` element.
+
+```js
+assert.exists(document.querySelector("head>link"));
+```
+
+Your `link` element should have `rel="stylesheet"` and `href="styles.css"`.
+
+```js
+const link = document.querySelector("head>link");
+assert.equal(link?.getAttribute("rel"), "stylesheet");
+assert.equal(link?.getAttribute("data-href"), "styles.css");
```

Comment:
```suggestion
const href = link?.getAttribute("data-href");
assert(href === "./styles.css" || href === "styles.css");
```

Reply:
```suggestion
const href = link?.getAttribute("data-href");
assert.match(href, /$(\.\/)?styles\.css^/);
```

why don't you do this for a single assert?